### ISSUE TITLE
Ignore model parameter when specifying table function

### ISSUE THREAD

**Issue by saeidakbari** [go-pg/pg#1397]
As go-pg uses ```db.Model()``` parameter (given model name) to create queries reference table, when ```Table()``` or ```TableExpr()``` is used, pg create two referenced table in queries and its lead to an error.
it would be good to have an ability to ignore given model name in ```db.Model()``` when user explicitly specified its corresponding table name

**Comment by vmihailenco** [comment]
Don't specify a model then, e.g. `db.Model().TableExpr("mytable").Select(&dst)`

**Comment by saeidakbari** [comment]
I know and i already use this for a while, but the problem is when passing ```nil``` to ```db.Model()``` many good features won't work,e.g `````WherePK()``` and some others

**Comment by vmihailenco** [comment]
Maybe this hack will work `db.Model().New().TableExpr("foo")` but it is not documented and I don't like it. If that is all you need I can add a readable API for it.

**Comment by saeidakbari** [comment]
Yes, of course this hack works but even though I don't like it, it's better to add a readable API and document for it.

**Comment by saeidakbari** [comment]
This hack somehow works but Even though I don't like it, it's better to add a readable API for it.
